scripts/docker_run

```diff
@@ -3,11 +3,19 @@
 set -o errexit
 set -o xtrace
 
-docker build --tag=oak .
+USERNAME=$(id -un)
+
+mkdir -p $PWD/bazel-cache
+touch $PWD/bazel-cache/.placeholder
+mkdir -p $PWD/cargo-cache
+touch $PWD/cargo-cache/.placeholder
+
+docker build --tag=oak --build-arg USERNAME="$USERNAME" --build-arg USERID="$(id -u)" --build-arg USERGRP="$(id -g)" .
```

Comment:
`--build-arg="USERNAME=$USERNAME"` (and below), and also one flag per line.

Reply:
Done